Microsoft.Toolkit.Uwp.UI.Behaviors/Focus/FocusBehavior.cs

```diff
@@ -92,24 +110,39 @@ private void ApplyFocus()
             }
 
             var focusedControlIndex = -1;
+            var listViewBaseControls = 0;
             for (var i = 0; i < Targets.Count; i++)
             {
                 var control = Targets[i].Control;
+                if (control is null)
+                {
+                    continue;
+                }
+
                 if (control.IsLoaded)
                 {
                     if (control.Focus(FocusState.Programmatic))
                     {
                         focusedControlIndex = i;
                         break;
                     }
+
+                    if (control is ListViewBase listViewBase)
+                    {
+                        // The list may not have any item yet, we wait until the first item is rendered.
+                        listViewBase.ContainerContentChanging -= OnContainerContentChanging;
+                        listViewBase.ContainerContentChanging += OnContainerContentChanging;
+                        listViewBaseControls++;
```

Comment:
It seems like a flag should be fine instead of counter.

Reply:
Thanks, I've push the fix in the new commit.